build_config/mono_pkg.yaml

```diff
@@ -6,6 +6,9 @@ stages:
     - group:
         - dartfmt: sdk
         - dartanalyzer: --fatal-infos --fatal-warnings .
+    - dartanalyzer: --fatal-infos --fatal-warnings .
+      dart:
+        - 2.0.0-dev.54
```

Comment:
Just bump this and the pubspec to 2.0.0

Reply:
We can do that separately, I want to have this tested at what is actually published

Reply:
ack